tools/packaging/release/release.sh

```diff
@@ -99,6 +99,26 @@ function _upload_vendored_code_tarball()
 	gh release upload "$RELEASE_VERSION" "$vendored_code_tarball"
 }
 
+function _upload_libseccomp_tarball()
+{
+	_check_required_env_var "GH_TOKEN"
+
+	[ -z "$RELEASE_VERSION" ] && RELEASE_VERSION=$(cat "$repo_root_dir/VERSION")
+
+	INSTALL_IN_GO_PATH=false $repo_root_dir/ci/install_yq.sh
+
+	versions_yaml="versions.yaml"
+	version=$(/usr/local/bin/yq read $versions_yaml "externals.libseccomp.version")
+	repo_url=$(/usr/local/bin/yq read $versions_yaml "externals.libseccomp.url")
+	download_url="$repo_url}releases/download/v$version"
+	tarball="libseccomp-$version.tar.gz"
+	asc="$tarball.asc"
+	curl -sSLO "$download_url/$tarball"
+	curl -sSLO "$download_url/$asc"
+	gh release upload "$RELEASE_VERSION" "$tarball"
+	gh release upload "$RELEASE_VERSIOB" "$asc"
```

Comment:
Optional suggestion: I personally prefer variables to have braces to avoid a brace expansion bear trap I've fallen into in the past and I find it clearer, as `${tarball}.asc` instead of `$tarball.asc`, but that might just be me.